pkg/envvar/env_var.go

```diff
@@ -0,0 +1,73 @@
+// envvar package helps converting name/value pairs to devfile EnvVars
+package envvar
+
+import (
+	"errors"
+	"strings"
+
+	devfilev1 "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+)
+
+// EnvVar represents an environment variable
+type EnvVar struct {
+	Name  string `yaml:"Name"`
+	Value string `yaml:"Value"`
+}
+
+// List represents a list of environment variables
+type List []EnvVar
+
+// ToDevfileEnv converts the EnvVarList to the list of Envs supported by devfile
+func (evl List) ToDevfileEnv() []devfilev1.EnvVar {
```

Comment:
```suggestion
func (evl List) ToDevfileEnvVar() []devfilev1.EnvVar {
```

I wonder if it makes more sense to rename the method to ToDevfileEnvVar since it is returning devfile EnvVar.